Framework/PythonInterface/plugins/algorithms/DNSFlippingRatioCorr.py

```diff
@@ -67,10 +68,10 @@ def _flipper_valid(self):
         """
         # sort workspaces for sf and nsf
         nsf = []
-        for key in self.input_workspaces.keys():
+        for key in list(self.input_workspaces.keys()):
```

Comment:
Should we add this to the recipes given here: http://www.mantidproject.org/Python_3? 2to3 adds `list()` everywhere it sees `keys()` and it is easy to miss that.